images/linux/scripts/installers/google-chrome.sh

```diff
@@ -42,6 +42,9 @@ download_with_retries $CHROME_DEB_URL "/tmp" "${CHROME_DEB_NAME}"
 apt install "/tmp/${CHROME_DEB_NAME}" -f
 echo "CHROME_BIN=/usr/bin/google-chrome" | tee -a /etc/environment
 
+# Remove Google Chrome repo
+rm -f /etc/cron.daily/google-chrome /etc/apt/sources.list.d/google-chrome.list /etc/apt/sources.list.d/google-chrome.list.save
```

Comment:
Should we document these apt source repo's?

Reply:
we don't use them anywere